extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.icons.js

```diff
@@ -0,0 +1,63 @@
+define(
+	'wikia.articleVideo.featuredVideo.jwplayer.icons',
+	[],
+	function () {
+		var icons = {
```

Comment:
I'd add a comments with names of icons. to easily find that these are icons from DS